src/orders/components/OrderReturnPage/OrderReturnRefundItemsCard/ReturnItemsCard.tsx

```diff
@@ -205,34 +200,32 @@ const ItemsCard: React.FC<OrderReturnRefundLinesCardProps> = ({
                     />
                   </TableCell>
                   <TableCell align="right">
-                    {isReturnable && (
-                      <TextField
-                        type="number"
-                        inputProps={{
-                          className: classes.quantityInnerInput,
-                          "data-test": "quantityInput",
-                          "data-test-id": id,
-                          max: lineQuantity.toString(),
-                          min: 0,
-                          style: { textAlign: "right" }
-                        }}
-                        fullWidth
-                        value={currentQuantity}
-                        onChange={handleChangeQuantity(id)}
-                        InputProps={{
-                          endAdornment: lineQuantity && (
-                            <div className={classes.remainingQuantity}>
-                              / {lineQuantity}
-                            </div>
-                          )
-                        }}
-                        error={isValueError}
-                        helperText={
-                          isValueError &&
-                          intl.formatMessage(messages.improperValue)
-                        }
-                      />
-                    )}
+                    <TextField
+                      type="number"
+                      inputProps={{
+                        className: classes.quantityInnerInput,
+                        "data-test": "quantityInput",
+                        "data-test-id": id,
```

Comment:
```suggestion
                        "data-test-id":"quantityInput",
```
Let's change it on the occasion. You'll also need to modify the file:
https://github.com/mirumee/saleor-dashboard/blob/3.0/cypress/elements/orders/order-refund.js#L2
```
  productsQuantityInput: '[data-test-id="quantityInput"]'
```